privacyidea/lib/token.py

```diff
@@ -453,6 +453,9 @@ def get_token_type(serial):
     :return: tokentype
     :rtype: string
     """
+    if serial and "*" in serial:
+        return None
```

Comment:
Oh, just realized we should probably return ``""`` and not None, because we return ``""`` if the token does not exist, and because we have specified the return type to be string.

Reply:
ok. done.